src/routes/formsgSiteCreation.ts

```diff
@@ -75,16 +77,18 @@ export class FormsgRouter {
         await this.sendCreateError(requesterEmail, repoName, submissionId, err)
         return res.sendStatus(200)
       }
-      const foundUser = await this.usersService.findByEmail(requesterEmail)
-      if (!foundUser) {
+      const foundRequester = await this.usersService.findByEmail(requesterEmail)
+      if (!foundRequester) {
         const err = `Form submitter ${requesterEmail} is not an Isomer user. Register an account for this user and try again.`
         await this.sendCreateError(requesterEmail, repoName, submissionId, err)
         return res.sendStatus(200)
       }
+      const foundOwner = await this.usersService.findOrCreateByEmail(ownerEmail)
```

Comment:
Do we need to handle the case where `foundOwner` is not defined here?

Reply:
Nope - foundOwner retrieves an existing user if one with that email exists in our db, and creates one otherwise, so a user is always returned